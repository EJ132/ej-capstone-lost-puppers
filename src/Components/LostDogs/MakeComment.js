import React from 'react'

export default function MakeComment(props){
    return (
        <li key={props.id}>
            <h4>{props.user_name}</h4>
            <p>{props.comment}</p>
            <p id="comment_date">{props.date_created}</p>
        </li>
    )
}