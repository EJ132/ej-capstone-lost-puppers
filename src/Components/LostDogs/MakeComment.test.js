import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import MakeComment from './MakeComment'

describe(`MakeComment component`, () => {

  it('renders a MakeComment page by default', () => {
    const wrapper = shallow(<MakeComment />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})
