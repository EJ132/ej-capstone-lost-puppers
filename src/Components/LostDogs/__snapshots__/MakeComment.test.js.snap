// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakeComment component renders a MakeComment page by default 1`] = `
<li>
  <h4 />
  <p />
  <p
    id="comment_date"
  />
</li>
`;
